### PR TITLE
[gatsbygram] use `graphql` imported from gatsby instead of global

### PR DESCRIPTION
**Pull request by tsriram**
This should get rid of the warnings when you run gatsbygram example.

### CODE REVIEW

**Comment by KyleAMathews** [comment]
Deploy preview for *using-postcss-sass* failed.

Built with commit 0384068a59eb64e9c1703b0d5a93b89f343a35d1

https://app.netlify.com/sites/using-postcss-sass/deploys/5b61adee1f12b779427ef2bc

**Comment by KyleAMathews** [comment]
Deploy preview for *image-processing* failed.

Built with commit 0384068a59eb64e9c1703b0d5a93b89f343a35d1

https://app.netlify.com/sites/image-processing/deploys/5b61adee1f12b779427ef2c2

File: examples/gatsbygram/src/components/modal.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ import * as PropTypes from "prop-types"
 import { push, StaticQuery } from "gatsby"
 
 import { rhythm } from "../utils/typography"
+import { graphql } from "gatsby"
```
Comment (pieh) [inline examples/gatsbygram/src/components/modal.js]: Please merge with line 9

File: examples/gatsbygram/src/components/post.js [hunk 2]
```diff
@@ -6,6 +6,7 @@ import Img from "gatsby-image"
 
 import { rhythm, scale } from "../utils/typography"
 import presets from "../utils/presets"
+import { graphql } from "gatsby"
```
Comment (pieh) [inline examples/gatsbygram/src/components/post.js]: Please merge with line 4

**Comment by KyleAMathews** [comment]
Deploy preview for *gatsbyjs* failed.

Built with commit 9041e3153ea5a18214d4521939d6a6b83e11c8f9

https://app.netlify.com/sites/gatsbyjs/deploys/5b61ee7cc6aed62d22762fe4

**Comment by m-allanson** [comment]
@tsriram I think @pieh meant to combine the `gatsby` imports onto one line. e.g.

```diff
- import { push, StaticQuery } from "gatsby"
- import { graphql } from "gatsby"
+ import { push, StaticQuery, graphql } from "gatsby"
```

**Comment by tsriram** [comment]
Ha! Sorry, looks like I was too sleepy when I made that change last night :man_facepalming:

**Comment by KyleAMathews** [comment]
Deploy preview for *using-contentful* failed.

Built with commit 54800eab5373e92ca6e4da3d9df8724d2ba431c8

https://app.netlify.com/sites/using-contentful/deploys/5b632c9afdd72a2fce83ae3f

**Comment by jquense** [comment]
We should merge that codemod I wrote :P

**Pull request merged**